public/steelthreads/asciidocs/en/creating-addresses.adoc

```diff
@@ -12,14 +12,18 @@
 // The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
 
 // TODO: figure out enmasse url
-:enmasse-url: https://console-enmasse.apps.pwright.openshiftworkshop.com/console/my-example-space
-// https://console-enmasse-my-example-space.apps.pwright.openshiftworkshop.com/#/dashboard
+:enmasse-url: https://console-enmasse.apps.city.openshiftworkshop.com/console/my-example-space
+// https://console-enmasse-my-example-space.apps.city.openshiftworkshop.com/#/dashboard
+
+// tag::intro[]
 
 = Creating addresses in EnMasse space
 
 You create two addresses in EnMasse to route messages to the Spring Boot app.
 
-. Log into link:{enmasse-url}[EnMasse, window="_blank"] by selecting the *OpenShift* button on the log in screen and accepting the permissions request.
+// end::intro[]
+
+. Log into link:{enmasse-url}[EnMasse] by selecting the *OpenShift* button on the log in screen and accepting the permissions request.
```

Comment:
can we add `window="_blank"` to all links? I believe the requirement for now was to open in new tabs...

so this would become:
```
link:{enmasse-url}[EnMasse, window="_blank"]
```
for example.

Reply:
that makes sense, will update adocs

Reply:
not sure why the verification asterisks  displays as shown. checked my preview and as you suggested, it's been overwritten by list-style-type: none; in css

Reply:
ie ul.checklist sets
list-style-type: none;